src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -1767,7 +1767,7 @@ private class ACCT_IndividualAccounts_TEST {
                 String FirstName1= 'ConFirstName ' + i;
                 String LastName = 'ConLastName ' + i;
 
-            	System.assertEquals(LastName + ' (' + FirstName1 + ') ' + ' Household', returnAccounts[i].Name);
+                System.assertEquals(LastName + ' (' + FirstName1 + ') ' + ' Household', returnAccounts[i].Name);
```

Comment:
@kdy618 same here with the order...

Reply:
@lifewithryan , on line 1770, FirstName1 is actually not calling/asking to return the 1st order fo the first name, FirstName1 is a variable.